README.md

```diff
@@ -122,9 +131,13 @@ an SAUTR that will be verified. By default "enableSautrCheck" will be false.
 
 All other fields must be provided.
 
+The businessVerificationCheck field allows the calling service to bypass the verification and
+continue to register where a successful match is found. By default "businessVerificationCheck" will be false.
+
 ```
 {
     "continueUrl" : "/test",
+    "businessVerificationCheck" : false ,
```

Comment:
I think these have been set to true by default, to not break the current Sole Trader flows